dataflows/processors/unwind.py

```diff
@@ -0,0 +1,39 @@
+from dataflows.helpers.resource_matcher import ResourceMatcher
+
+
+def unwind(from_key, to_key, transformer=None, resources=None, source_delete=True):
```

Comment:
I think we should also add here the 'to_key' field (with its type and other options)
(or - if `from_key == to_key` - update the field's type accordingly)